source/moto_payments/index.html.md.erb

```diff
@@ -16,43 +16,68 @@ This means you can take payments:
+You can set up MOTO payments on either a test account or a live account.
 
-[Contact Government Banking](https://www.gov.uk/government/groups/government-banking-service-gbs) instead if you use them.
+MOTO payment screens are the same as online payment screens, excluding the following:
 
-## Set up your account
+- MOTO payments screens do not have a section for billing address
+- [Strong Customer Authentication](/set_up_3dsecure/#set-up-strong-customer-authentication-3ds1-and-3ds2) does not apply to MOTO payments
 
-Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) and we'll:
+## Set up MOTO payments on a test account
 
-- review your [Payment Card Industry Data Security Standards (PCI DSS) assessment](/security/#payment-card-industry-pci-compliance) to check you can take payments over the phone and by post
-- enable MOTO payments in your service
+[Contact GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk) to ask us to enable MOTO payments on your test account.
 
-## Create a MOTO payment
+You do not need any provide us with any supporting information.
+
+## Set up MOTO payments on a live account
+
+How you set up MOTO payments on a live account differs depending on what payment service provider (PSP) you use.
+
+### Set up MOTO payments on a live account with Worldpay or ePDQ
+
+If you use Worldpay or ePDQ, you must:
+
+- use a service for MOTO payments that is separate to your online payments service
+- process MOTO payments on a specific MOTO merchant code
+
+1. Create a separate MOTO payments service if you do not have one already by signing in to your [GOV.UK Pay account](https://selfservice.payments.service.gov.uk/) and going to __My services__. Select __Add a new service__ to add a new service for MOTO payments to your account.
+
+1. If you do not already have a MOTO merchant code, ask for a new code by contacting your PSP or Government Banking as required. You will use this merchant code when you [connect your new service to your PSP](/switching_to_live/#go-live).
+
+1. Send your [PCI DSS Self Assessment Questionnaire](https://www.pcisecuritystandards.org/document_library?category=saqs) and service name to [GOV.UK Pay support](mailto:govuk-pay-support@digital.cabinet-office.gov.uk).
 
-Make sure you've collected your user's:
+We will review your PCI DSS compliance. Your PCI DSS compliance requirements are different for MOTO payments compared to online payments.
 
-- card number
-- card expiry date
-- card security code
-- name from their card
-- email address - if you’ve [enabled collecting your users’ email addresses](/integrate_with_govuk_pay/#collecting-your-users-39-billing-addresses)
+When we are sure that you have met your PCI DSS compliance requirements, we enable MOTO payments in your service. You can then take MOTO payments.
+
+### Set up MOTO payments on a live account with Stripe
+
+Stripe is GOV.UK Pay’s current PSP.
```

Comment:
we can probably also delete this line to match the first section, but also happy to leave in

Reply:
yes